docs/dev/stream/state/state.md

```diff
@@ -471,15 +471,12 @@ e.g. after restart from savepoint.
 
 ##### Cleanup during RocksDB compaction
 
-If RocksDB state backend is used, another cleanup strategy is to activate Flink specific compaction filter.
+If RocksDB state backend is used, another cleanup strategy is to call Flink specific compaction filter.
```

Comment:
Currently it sounds like that the cleanup strategy won't be active by default or there are other strategies that can be used – Maybe that's just me. My proposal would be:

> If the RocksDB state backend is used, a Flink specific compaction filter will be employed to enable background cleanup.